kasparovsyncd/mqtt/selected_parent_chain.go

```diff
@@ -0,0 +1,32 @@
+package mqtt
+
+import (
+	"github.com/kaspanet/kaspad/rpcmodel"
+	"github.com/kaspanet/kasparov/apimodels"
+)
+
+// SelectedParentChainTopic is an MQTT topic for changes in the
+// selected parent chain
+const SelectedParentChainTopic = "selected-parent-chain"
+
+// PublishSelectedParentChainNotifications publishes notifications for changes in the selected parent chain
+func PublishSelectedParentChainNotifications(removedChainHashes []string, addedChainBlocks []rpcmodel.ChainBlock) error {
+	if !isConnected() {
+		return nil
+	}
+
+	notificationData := &apimodels.SelectedParentChainNotification{
+		AddedBlockHashes:  make([]string, len(addedChainBlocks)),
+		RemoveBlockHashes: make([]string, len(removedChainHashes)),
+	}
+
+	for i, block := range addedChainBlocks {
+		notificationData.AddedBlockHashes[i] = block.Hash
+	}
+
+	for i, hash := range removedChainHashes {
```

Comment:
Why not `notificationData.RemoveBlockHashes = removedChainHashes`?

In case there's a good reason - you can use `copy` here.
On the other hand, maybe you want to keep things as is to look similar to how you addign AddedBlockHashes - your call